lib/outputs/outputs.py

```diff
@@ -3,20 +3,19 @@
 from colorama import Fore, Style
 
 from config.config import csv_file_suffix
-from lib.check.models import Organizations_Info
 from lib.outputs.models import Check_Output_CSV
 from lib.utils.utils import file_exists, open_file
 
 
-def report(check_findings, output_options, audit_info, organizations_info):
```

Comment:
Why don't you include `organizations_info` within `audit_info`? Organizations are a feature of AWS and we don't want to couple our common logic with a cloud vendor. Maybe we can include it there, what's your opinion?

Reply:
Agree